include/RAJA/index/IndexSet.hpp

```diff
@@ -90,7 +90,12 @@ class TypedIndexSet<T0, TREST...> : public TypedIndexSet<TREST...>
                 "All segments must have the same value_type");
 
   //! Construct empty index set
+#ifdef _WIN32
```

Comment:
Can this be tested on 2017 and/or guarded for that specific version?

Reply:
Yes, it can.  It works on at least some version of VS2017 or it wouldn't have passed appveyor.

I'd much rather this check for a low _MSC_VER: https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=vs-2019
Or, even better, add a define somewhere for the bug and/or issue that does the check which is then used here so we can track the workaround.

Reply:
I've changed this to _MSC_VER < 1910 and adjusted the comment as appropriate.